plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/messages.properties

```diff
@@ -48,3 +49,6 @@ project.filter.hint=Filter projects by name or ID
 error.file.does.not.exist={0} does not exist.
 error.is.a.directory={0} is a directory.
 error.is.not.readable={0} is not readable.
+service.key.created=A service key for the App Engine default service account created and set:\n{0}
```

Comment:
"Created a service account key for the App Engine app default service account"
(same language as Pantheon)